aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java

```diff
@@ -69,33 +80,47 @@ public void run(String... args) {
             options.addOption("c", true, "aggregation overrides to be applied - xml");
 
             CommandLineParser parser = new DefaultParser();
-            CommandLine line = parser.parse(options, args);
-
-            URL url = new URL(line.getArgs()[0]);
-            URLConnection conn = url.openConnection();
-
-            List<String> inputFiles = new ArrayList<>();
-
-            try (BufferedReader reader = new BufferedReader(new InputStreamReader(conn.getInputStream(), StandardCharsets.UTF_8))) {
-                inputFiles = reader.lines().collect(Collectors.toList());
+            CommandLine commandLine = parser.parse(options, args);
+
+            //  Check parameters:
+            //      Expecting the following params -
+            //        - layerName
+            //        - subset
+            //        - outputFile
+
+            String layerName = commandLine.getArgs()[0];
+            String subset = commandLine.getArgs()[1];
+            String destinationFile = commandLine.getArgs()[2];
```

Comment:
Probably can determine the output file location here using config and the job id at some point.

Reply:
Yep - is the target location for the aggregated file the same location as the status file?  Or will we have another base location for the aggregated file?

Reply:
Can be in the same folder - but the status file is separate to the output file